libfuse/dir.go

```diff
@@ -343,6 +343,12 @@ func (d *Dir) Attr(ctx context.Context, a *fuse.Attr) (err error) {
 	d.folder.fs.log.CDebugf(ctx, "Dir Attr")
 	defer func() { d.folder.reportErr(ctx, libkbfs.ReadMode, err) }()
 
+	err = libkbfs.EnableDelayedCancellationWithGracePeriod(
```

Comment:
I think we need some comments here explaining why these particular file system calls need delayed cancellation.  Actually i forget the exact reason why.  Maybe POSIX says that they should always return successfully even when an interrupt/SIGALRM is received, and some programs like git rely on that?